doc/whats_new.rst

```diff
@@ -56,6 +56,16 @@ Enhancements
     generated during build. Distribution packages will still contain generated
     C/C++ files. By `Arthur Mensch`_
 
+    - Enhanced MLP with Adams, Nesterov's momentum, and early stopping (`#5214 <https://github.com/scikit-learn/scikit-learn/pull/5214>`_)
+
+    - Enhanced with colorblind compatibility to various examples by `Johanna Hansen`_
+
+    - Made scaling a sparse matrix along axis 0 to accept a CSR format by default by `Manoj Kumar`_
```

Comment:
It would be great to add links to the issue for all the added features. For the colorblindness, maybe just link to the original issue.